src/mds/CInode.cc

```diff
@@ -4027,21 +4027,39 @@ void CInode::decode_import(bufferlist::const_iterator& p,
 
 void InodeStoreBase::dump(Formatter *f) const
 {
+  f->open_object_section("inode");
   inode.dump(f);
+  f->close_section();
   f->dump_string("symlink", symlink);
+
+  f->open_array_section("xattrs");
+  for (const auto &p : xattrs) {
```

Comment:
use structured binding: `for (const auto& [key, val] : xattrs) {`

Reply:
ok